build_tools/ci/run_matmul_test.sh

```diff
@@ -195,3 +227,16 @@ run_matmul_test \
     --peano_install_path "${PEANO}" \
     --mlir_aie_install_path "${MLIR_AIE_INSTALL}" \
     --vitis_path  "${VITIS}" \
+    --pipeline "pad"
+
+run_matmul_test \
+    --name "matmul_i32_i32_small_amd-aie_xrt_simple-pack" \
+    --lhs_rhs_type "i32" \
+    --acc_type "i32" \
+    --shapes "small" \
+    --target_backend "amd-aie" \
+    --device "xrt" \
+    --peano_install_path "${PEANO}" \
+    --mlir_aie_install_path "${MLIR_AIE_INSTALL}" \
+    --vitis_path  "${VITIS}" \
+    --pipeline "simple-pack"
```

Comment:
lets also track, simple-pack large?

Reply:
I'm not sure if `M=64, N=64, K=16` works since I've majorly tested with `M=8, N=*, K=*`.

I'll add and check it directly on CI now. :P

Reply:
Thats what it was testing before this PR though, since simple pack is default and we had a large test?

Reply:
Aah! It worked! :')